src/QsCompiler/Tests.Compiler/Tests.Compiler.fsproj

```diff
@@ -150,9 +157,6 @@
 
   <ItemGroup>
     <ProjectReference Include="..\CommandLineTool\CommandLineTool.csproj" />
-    <ProjectReference Include="..\TestTargets\Simulation\Example\Example.csproj">
```

Comment:
Looks like this has been lost on accident. That project reference would need to be added back.